tests/core/htmldataprocessor.js

```diff
@@ -1328,9 +1328,11 @@
 
 	// The `src="&#10;&#106;javascript:..."` is treated as some different protocol in Edge/IE8 and also Chrome on Linux.
 	// IE8 treats it as an URL and opens it which reloads the whole page.
-	// While on Edge the tests passes, the browser prompts with "open as" dialog.
+	// While on Edge and on IE11 the tests passes, the browser prompts with "open as" dialog.
 	// On Chrome Linux it prompts `Open xdg-open?` dialog, which gains focus so other tests requiring focus fails.
-	if ( !( CKEDITOR.env.ie && CKEDITOR.env.version == 8 || CKEDITOR.env.edge ) && !( CKEDITOR.env.chrome && bender.tools.env.linux ) ) {
+	if ( !( CKEDITOR.env.ie && ( CKEDITOR.env.version == 8 || CKEDITOR.env.version == 11 ) ) &&
```

Comment:
This was an issue only in IE11 on Windows 10, can we narrow the `if` condition? (I know there is no env variable checking for Windows environment, if it is easy enough to detect we can add it to `bender.tools.env` - if not leave it as is).